### PR TITLE
[INLONG-3489][K8s] Improve documentation in helm chart

### PR DESCRIPTION
**Pull request by shink**
Fixes: #3489

### Motivation

- provide some useful information in [NOTES.txt](docker/kubernetes/templates/NOTES.txt)
- provide more detailed comments in [values.yaml](docker/kubernetes/values.yaml) to help users configure it

### Modifications

The [NOTES.txt](docker/kubernetes/templates/NOTES.txt) content will be printed when installing the chart.

image

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### CODE REVIEW

File: docker/kubernetes/README.md [hunk 1]
```diff
@@ -1,21 +1,65 @@
...
+
+```shell
+export DASHBOARD_SERVICE_IP=$(kubectl get svc inlong-dashboard --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}"  -n inlong)
+```
+
+And then access `http://$DASHBOARD_SERVICE_IP:30080`
+
+> NOTE: It may take a few minutes for the `LoadBalancer` IP to be available. You can check the status by running `kubectl get svc inlong-dashboard -n inlong -w`
+
+The default username and password are `inlong`. You can access the InLong Dashboard through them.
```
Comment (shink) [inline docker/kubernetes/README.md:60]: @dockerzhang Are the default username password right?

**Review by healchow — APPROVED** [review]

File: docker/kubernetes/README.md (hunk 1 above)
Comment (dockerzhang) [reply docker/kubernetes/README.md:60]: `admin`
`inlong`

**Comment by dockerzhang** [comment]
@shink please update the document on the [office website](https://inlong.apache.org/docs/next/deployment/k8s).
you can refer to https://github.com/apache/incubator-inlong-website/pull/342

File: docker/kubernetes/README.md (hunk 1 above)
Comment (shink) [reply docker/kubernetes/README.md]: Done.

**Comment by shink** [comment]
@dockerzhang OK.

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**